deluge/core/daemon_entry.py

```diff
@@ -17,6 +17,7 @@
 import deluge.configmanager
 import deluge.error
 from deluge.ui.baseargparser import BaseArgParser
+from deluge.ui.util import lang
```

Comment:
I can't see any other use for it in `lang.py` so perhaps move it here is a good idea